backend/mongo/mongodb.go

```diff
@@ -181,7 +181,7 @@ func (s *SmartCollection) Collection() *mgo.Collection {
 
 func (s *SmartCollection) EnsureIndexes(idxs []*mgo.Index) error {
 	for _, idx := range idxs {
-		s.log.Debugf("Ensuring index: %s", idx.Name)
+		s.log.Infof("Ensuring index: %s", idx.Name)
```

Comment:
Has this been done only on a start-up/init? Otherwise, it looks like Debug for me.

Reply:
From the looks of it, it is similar to the MySQL backend where it tries to create the db and table. Since the lib is preforming a mutation, I felt that a client would want to know that. Happy to change these back to `Debug` (as well as the MySQL bits) if you think otherwise.

At the end of the day I was going for consistency between all the backends 😎